news/2020-04-13-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "osu! Spotlights: Recruiting New Team"
+date: 2020-04-13 08:00:00 +0000
+---
+
+Following our latest Spotlight season released very recently, we are underway reworking osu! Spotlights from the ground up, starting with recruiting for a brand new team and adding a whole new league system in future seasons.
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving the curation and participation in osu! Spotlights a way better reason. While recommending beatmaps to you, a new competitive and seasonal league closely attached to the chosen beatmaps is going to be added. With this, you and your friends will have even more reason to delve deep into the best creations osu! has to offer.
```

Comment:
```suggestion
Since March 2017, the osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal and showcased for their excellence, we want to expand on the idea by making our choices meaningful to both players and mappers alike. 

As such, we intend to create **a new competitive and seasonal league** based on spotlighted maps. With this, you and your friends will have even more reason to delve into some of the best creations that osu! has to offer.
```